tree/tree/inc/TTree.h

```diff
@@ -642,6 +642,12 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
    virtual void            SetObject(const char* name, const char* title);
    virtual void            SetParallelUnzip(bool opt=true, Float_t RelSize=-1);
    virtual void            SetPerfStats(TVirtualPerfStats* perf);
+   /**
+    * \brief Sets the maximum number of rows to be shown before `<CR>` when calling Scan().
+    * \param n the maximum number of rows. Default=50, if 0, all rows of the Tree are shown
+    * and there is no need to press `<CR>` or quit to exit the function.
```

Comment:
I think there is no need to explain the extra details
```suggestion
    * \param n the maximum number of rows. Default=50, if 0, all rows of the Tree are shown.
```

Reply:
To me, it seemed important to explain about the interactivity.

If you use Scan(50) with root -b, your script will hang until you press a key, If you use Scan(0), your script will exit and thus it can be run in batch mode without problems.